README.md

```diff
@@ -2,22 +2,82 @@
 
 This repo contains experiments that will comprise a walking skeleton of the Libero publishing system.
 
-Execute `./example.sh` in your terminal to follow a demo.
+Execute `./example.sh` in your terminal to follow a demo. This shows:
 
-It contains:
+1. Creating an article
+2. Adding a new version
+3. Updating a version
+4. Deleting an article
 
 ## Applications
 
 These can be run using `docker-compose up -V`.
 
-- `api-dummy`: A dummy implementation of the API, containing sample XML content in multiple languages from eLife, Hindawi, SciELO, International Journal of Microsimulation and Wikipedia, and corresponding RELAX NG schemas for each source. Can be viewed at http://localhost:8081/articles.
-  - XML content is stored in `data`
-  - Schemas are stored in `public/schemas`
-- `dashboard`: A basic implementation of the dashboard, that lists all events passed on the event bus in multiple languages. Can be viewed at http://localhost:8082/.
-- `journal`: A basic implementation of the public-facing site, that lists and displays articles from an API implementation in multiple languages. Can be viewed at http://localhost:8080/.
+### `api-doc`
+
+Can be viewed through the `api-gateway` at http://localhost:8085/.
+
+Contains an example RAML definition of a Libero API: shows running two article stores (with different content schemas), including adding custom extensions to article store endpoints.
+
+### `api-gateway`
+
+Contains a simple Nginx API gateway.
+
+* http://localhost:8085/ proxies `api-doc`
+* http://localhost:8085/articles proxies `article-store`
+
+### `article-store`
+
+Can be viewed through the `api-gateway` at http://localhost:8085/articles.
+
+Contains an article store 
```

Comment:
Perhaps mention Django here as other frameworks/technologies have been mentioned on the other services.